### PR TITLE
feat: add new useful repository for pentest (Cognito Scanner)

### PR DESCRIPTION
**Pull request by cfpadok**
Cognito Scanner is a CLI open-source tool. It enables to pentest Cognito AWS instance easily. It can be installed through a python package

### CODE REVIEW

**Review by ThreatLEVELred — APPROVED** [review]

**Comment by cat998** [comment]
已收到！

**Review by ThreatLEVELred — APPROVED** [review]

**Review by ThreatLEVELred — APPROVED** [review]

**Review by AnadyaNair — APPROVED** [review]

**Review by jarfam — COMMENTED** [review]
1

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Repository | Description
 [API Security Checklist](https://github.com/shieldfy/API-Security-Checklist) | Checklist of the most important security countermeasures when designing, testing, and releasing your API
 [APT Notes](https://github.com/kbandla/APTnotes) 									| Various public documents, whitepapers and articles about APT campaigns
 [Bug Bounty Reference](https://github.com/ngalongc/bug-bounty-reference) 			| List of bug bounty write-up that is categorized by the bug nature
+[Cognito Scanner](https://github.com/padok-team/cognito-scanner) | CLI tool to pentest Cognito AWS instance
```
[the remark this answers is not in the document]
Comment (jarfam) [reply README.md:67]: 1

**Review by Cloneface — COMMENTED** [review]
Haw you can doing that

**Comment by jarfam** [comment]
برای ماست همه چیز

**Review by jarfam — COMMENTED** [review]
wow

**Review by Cookiesboy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (merlinwizzard) [inline README.md:67]: check all wallets for balance virus..

**Review by merlinwizzard — APPROVED** [review]

**Comment by Chan9390** [comment]
Closing the PR as its not an awesome list.

**Pull request closed without merging**